### ISSUE TITLE
Add TaskInstance* properties to JobFlowInstanceConfig in emr.py

### ISSUE THREAD

**Issue by avmarchenko** [cloudtools/troposphere#2017]
**Ask**
Hello, I'm looking to get the two [TaskInstance*](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-elasticmapreduce-cluster-jobflowinstancesconfig.html) properties to the `JobFlowInstanceConfig` resource in emr.py. First time posting so let me know if I missed some protocol or step or if you have any questions.

**Troubleshooting steps I took**
Following the info in CONTRIBUTING.md, I constructed the spec, generated emr.py and checked the diff to see if the properties appeared. They did not. I then checked the `CloudFormationResourceSpecification.json` file for the presence of those two properties and they were not there. I guess this means that AWS hasn't updated this content on their end?

**Comment by markpeek** [comment]
@avmarchenko I need to update CONTRIBUTING based on the recent changes to auto-generate from the specification file. We now have a github action that runs daily to look for updates to the file and creates a PR automatically. As you point out, these properties are not in the specification files yet but hopefully will show up soon.

**Comment by avmarchenko** [comment]
Thanks for the quick reply @markpeek ! As a temporary workaround does the following work as a hacky runtime extension or will there be some silent failure (obvious to you but not me) I should be aware of?
```
from troposphere.emr import JobFlowInstancesConfig, InstanceFleetConfigProperty, InstanceGroupConfigProperty
JobFlowInstancesConfig.props["TaskInstanceFleets"] = (InstanceFleetConfigProperty, False)
JobFlowInstancesConfig.props["TaskInstanceGroup"] = (InstanceGroupConfigProperty, False)

...code that uses JobFlowInstancesConfig...
```

Should I close this issue?

**Comment by markpeek** [comment]
Yes, that technique is an easy localized workaround. The needed change is to make it a list of those objects (according to the docs):
```
from troposphere.emr import JobFlowInstancesConfig, InstanceFleetConfigProperty, InstanceGroupConfigProperty
JobFlowInstancesConfig.props["TaskInstanceFleets"] = ([InstanceFleetConfigProperty], False)
JobFlowInstancesConfig.props["TaskInstanceGroup"] = ([InstanceGroupConfigProperty], False)
```
Leave this open for now so I can track it and feel free to follow up if the above works.

**Comment by avmarchenko** [comment]
Thank you!

**Comment by markpeek** [comment]
@avmarchenko I patched these changes into Release 4.0.0 via 2c94e3dd9d8a516762a267a98216fd0a43365908

**Comment by avmarchenko** [comment]
Thank you!
